### ISSUE TITLE
[BUG] - Excessive nose up trim required on final when handflying

### ISSUE THREAD

**Issue by OD-Emperor** [invernyx/md11-bugs#1474]
### Describe the bug

When disengaging AP there is a high amount of nose up trim required on final to remain stable

### Steps to reproduce

Load the MD-11 GE F normally, fly any distance, and land using an ILS. On the ILS you will notice the AP commands a normal pitch trim. In the video supplied, you can see it's at about 9.0 on the proper VAPP speed. 

AP is disengaged at 18 seconds and you can see it briefly flies normally and the position of the yoke indicates it was not trying to input any force on the yoke to maintain the approach path. A few seconds later and especially once I start to add inputs to the yoke it very quickly loses the glideslope when it shouldn't.

During this time I have to increase the nose up trim excessively to about 13.9 when the speed and wind conditions have not changed by a large amount.

In some instances I have had to trim to the max available deflection at 15.5 to control the aircraft.

This behaviour was not experienced prior to .62 and is still in .64 (where the video is from). I suspect it has something to do with the improved rotation behaviour.

This has happened now on 4 different flights in the MD11F GE, and I'm unsure if it translates to the other variants but I'm very sure it will considering they're all quite similar. 

Video flight route/weight:

**ROUTE:
KIND/05L MAREO5 BDOCK DCT DJB DCT JHW J70 LVZ LENDY8 KJFK/31R**

 WEIGHTS
                               -------

                EST      MAX     ACTUAL

PAX            4              ......

CARGO      81.1              ......

PAYLOAD   81.4              ......

ZFW           198.0    204.7     ......

FUEL           25.3     35.0     ......  POSS EXTRA 9.7

TOW           222.4    232.1  LDG......

LAW           208.7    218.4     ......

------------------------------------------------------------------------------------

Also experienced this behaviour on the following flights:

NZAA/05R DCT PEBLU N759 GEMAC Q302 LIZZI DCT YMML/16

 WEIGHTS

                               -------

                   EST      MAX     ACTUAL

PAX            3          ......

CARGO      78.3      ......

PAYLOAD   78.5      ......

ZFW           195.1    204.7     ......

FUEL           43.5     52.9     ......  POSS EXTRA 9.4

TOW           237.7    247.1  LDG......

LAW           209.0    218.4     ......

------------------------------------------------------------------------------------

PHNL/04R OPIHI3 CARRP A579 CROBY DCT 05N168W 00N171W 05S174W 10S177W
15S180E DCT IGAPI DCT 25S177E DCT NISET A578 KALAG EXOPI8D NZAA/05R

              WEIGHTS

                               -------

                EST      MAX     ACTUAL

PAX          4              ......

CARGO    75.4              ......

PAYLOAD 75.8              ......

ZFW        192.3    204.7     ......

FUEL       92.3     92.3     ......  POSS EXTRA 0.0

TOW       283.7    283.7  TOW......

LAW           207.0    218.4     ......

------------------------------------------------------------------------------------

KTPA/19R ENDED2 LACEN DCT BRUTS Q118 THRSR Q79 SWAPP DCT HAGAL
GIIBS3 KIND/23R

     WEIGHTS

                               -------

                EST      MAX     ACTUAL

PAX          3              ......

CARGO    72.0              ......

PAYLOAD 72.2              ......

ZFW        188.8    204.7     ......

FUEL        29.6     46.6     ......  POSS EXTRA 17.1

TOW       217.4    234.5  LDG......

LAW           201.3    218.4     ......

### Expected behaviour

To fly using a reasonable amount of pitch trim not the max amount.

### Evidence

https://youtu.be/OujaBcTUbPc

### Simulator

Microsoft Flight Simulator 2020 (MSFS)

### Agreement

- [X] Yes

**Comment by joon44** [comment]
I agree. I also have very high trim settings above 12 units on short final

**Comment by KJA1582** [comment]
I concur, 15.2 up to 15.4 ANU on final

ZFW: 189648
LW: 201.8 at touchdown
Fuel: 12198 at touchdown

I do note that going manual, at LSAS release, the aircraft had no pitch moment other than what was used to release LSAS.

At the Gate, reentry of INIT data gave ZFWCG of 27.5, TOCG of 26.7, STAB TRIM at No Flex, F28, 0/0 Slope/Wind, 19C was hence calculated as 10.8

**Comment by Topofdescent5** [comment]
Same here, every flight I can remember in the last few weeks I'm at almost or full back trim and pulling on the yoke just to stay away from 3 or 4 red PAPIs. I'm at the point of not flying it anymore, sadly

**Comment by joon44** [comment]
No more a problem
